README.rst

```diff
@@ -4,7 +4,7 @@
     :target: https://pypi.org/project/folium
     :alt: PyPI Package
 
-.. |Travis| image:: https://travis-ci.org/python-visualization/folium.svg?branch=master
+.. |Travis| image:: https://travis-ci.org/python-visualization/folium.svg?branch=main
```

Comment:
Actually, this badge should probably be removed or replaced by a GitHub actions badge, as Travis is no longer used.
I'd tend to do this in a separate PR, but if you prefer I can update the PR to include that change as well

Reply:
Do you want to send a PR to add the GHA badge? We still use Travis-CI to build the docs but you can remove the badge in the PR.